src/transformStateWithClones.js

```diff
@@ -7,7 +7,50 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const ADD_PROPERTY_TYPE = 'addProperties';
+  const REMOVE_PROPERTY_TYPE = 'removeProperties';
+  const CLEAR_TYPE = 'clear';
```

Comment:
Type names are quite informative and do not require separate variables for renaming